bootstrapvz/plugins/file_copy/manifest-schema.yml

```diff
@@ -18,13 +18,12 @@ properties:
                 group:
                   type: string
               required: [dir]
-              additionalProperties: false
```

Comment:
TL;DR: Dont drink and code

I believe this is a residue from something else I've been working on, sorry about that, fixed